ui/lib/sync/addon/components/secrets/page/destinations.ts

```diff
@@ -94,11 +94,11 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
   async onDelete(destination: SyncDestinationModel) {
     try {
       const { name } = destination;
-      const message = `Successfully deleted destination ${name}.`;
+      const message = `Destination ${name} has been queued for deletion.`;
       await destination.destroyRecord();
       this.store.clearDataset('sync/destination');
       this.router.transitionTo('vault.cluster.sync.secrets.destinations');
```

Comment:
I wonder if it would be beneficial to transition to the destination details route now instead since that's where the purge information is?

Reply:
Yep! I asked Yankun about this last night. Just waiting for confirmation.

Reply:
Updated! Transitioned to the associations because then users can see the unsync status of each association